system-addon/common/Reducers.jsm

```diff
@@ -3,8 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
 
-const {actionTypes: at} = Components.utils.import("resource://activity-stream/common/Actions.jsm", {});
-const {Dedupe} = Components.utils.import("resource://activity-stream/common/Dedupe.jsm", {});
+const {actionTypes: at} = ChromeUtils.import("resource://activity-stream/common/Actions.jsm", {});
+const {Dedupe} = ChromeUtils.import("resource://activity-stream/common/Dedupe.jsm", {});
```

Comment:
@k88hudson I'm guessing we need to get both of these updated to support `ChromeUtils.import`?
https://www.npmjs.com/package/babel-plugin-jsm-to-commonjs
https://www.npmjs.com/package/babel-plugin-jsm-to-esmodules

Reply:
@k88hudson sometimes I amaze myself with such hacky hackery ;) ;) ;)
```diff
diff --git a/plugin.js b/plugin.js
index cd268e0..74d8a1b 100644
--- a/plugin.js
+++ b/plugin.js
@@ -110,16 +110,17 @@ module.exports = function plugin(babel) {
             init.get("property").isIdentifier() &&
             init.get("property").node.name === "utils"
           ) {
             results.push(id.node.name);
           }
         }
       });
     });
+    results.push("ChromeUtils");
     return results;
   }
 
   function replaceImports(nodes, ComponentNames, CuNames, basePath, replacePath) {
     nodes.forEach(p => {
       if (!p.isVariableDeclaration()) return;
       p.traverse({
         CallExpression(path) {
@@ -156,18 +157,23 @@ module.exports = function plugin(babel) {
 
   }
 
   function replaceXPCOM(paths, basePath, replacePath) {
     paths.forEach(path => {
       if (
         path.isExpressionStatement() &&
         path.get("expression").isCallExpression() &&
+        ((
         path.get("expression.callee.object.name").node === "XPCOMUtils" &&
         path.get("expression.callee.property.name").node === "defineLazyModuleGetter"
+        ) || (
+        path.get("expression.callee.object.name").node === "ChromeUtils" &&
+        path.get("expression.callee.property.name").node === "defineModuleGetter"
+        ))
       ) {
         const argPaths = path.get("expression.arguments");
         const idName = argPaths[1].node.value;
         let filePath = argPaths[2].node.value;
 
         if (!filePath.match(basePath)) return;
 
         if (replacePath) filePath = filePath.replace(basePath, "");
```